Detectors/ITSMFT/ITS/workflow/include/ITSWorkflow/ClustererSpec.h

```diff
@@ -42,9 +42,13 @@ class ClustererDPL : public Task
   void run(ProcessingContext& pc) final;
 
  private:
+  void updateTimeDependentParams(ProcessingContext& pc);
+  void finaliseCCDB(ConcreteDataMatcher& matcher, void* obj);
```

Comment:
```suggestion
  void finaliseCCDB(ConcreteDataMatcher& matcher, void* obj) final;
```

Reply:
Don't you need the final / override?

Reply:
@ktf: I do, but adding it does not help, I have also run with the `debug` in https://github.com/AliceO2Group/AliceO2/blob/d4de430a8c943a7de54587456e4b5c46b97c13f0/Framework/Core/include/Framework/Task.h#L104 changed to `info`, it is not invoked.